File: Classes/Work.py
class Work():

    def __init__(self, id: int,  name :str, levelRequired: int, expReward: int, moneyReward: int):
        self.id = id
        self.name = name
        self.levelRequired = levelRequired
        self.expReward = expReward
        self.moneyReward = moneyReward

#Имя работы должно начинаться с глагола в инфинитиве
Works : list[Work] = [
    Work( 
        0,
        'Учить детей питону в шараге',
        0,
        50,
        40
     ),
     Work( 
        1,
        'Подрабатывать у Задорожного плюсовиком',
        4,
        120,
        80
     ),
     Work( 
        2,
        'Формашлепить стажировские задания',
        8,
        150,
        110
     ),
     Work( 
        3,
        'Ревьюить код шушпенчиков',
        12,
        180,
        130
     ),
     Work( 
        4,
        'Потаунтить Ривендера',
        18,
        200,
        180
     )
]   

File: backend/src/main/java/ru/gamebot/backend/models/Work.java
package ru.gamebot.backend.models;

import jakarta.persistence.Entity;
import jakarta.persistence.GeneratedValue;
import jakarta.persistence.GenerationType;
import jakarta.persistence.Id;
import lombok.Data;

@Data
@Entity
public class Work {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Integer id;

    private String name;
    private Integer levelRequired;
    private Integer expReward;
    private Integer moneyReward;

}
